test/any/datastore/rate_limited_DAO-test.es6.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('RateLimitedDAO', function() {
+  let rateLimitedDAO;
+  foam.CLASS({
+    package: 'test.RateLimitedDAO',
+    name: 'item',
+    properties: ['id'],
+  });
+  beforeEach(function() {
+    rateLimitedDAO = org.chromium.apis.web.RateLimitedDAO.create({
+      interval: 100,
```

Comment:
Can we make this be `50` or less without seeing test flake? Don't want these tests to take too long to run.